lighthouse-core/lib/log.js

```diff
@@ -40,8 +40,9 @@ class Emitter extends EventEmitter {
    * Fires off all status updates. Listen with
    * `require('lib/log').events.addListener('status', callback)`
    * @param {string} title
+   * @param {...*} args
    */
-  issueStatus(title, args) {
+  issueStatus(title, ...args) {
     if (title === 'status' || title === 'statusEnd') {
       this.emit(title, args);
```

Comment:
wouldn't this stick all current args in an array? will that impact anything outside of changed below?